### PR TITLE
dockerized the app

### PR DESCRIPTION
**Pull request by olgaszatkowska**

### CODE REVIEW

File: backend/media_server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN chmod +x /entrypoint.sh
+
+# Add custom nginx.conf
+COPY ./media_server/nginx.conf /etc/nginx/nginx.conf
+
+# Expose port
+EXPOSE 80
+# Start nginx
+ENTRYPOINT ["/entrypoint.sh"]
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (CreativePhilip) [inline backend/media_server/Dockerfile]: Do we need this ?

File: backend/requirements/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pre-commit==2.21.0  # https://github.com/pre-commit/pre-commit
+
+# Django
+# ------------------------------------------------------------------------------
+factory-boy==3.2.1  # https://github.com/FactoryBoy/factory_boy
+model-bakery==1.9.0  # https://github.com/model-bakers/model_bakery
+django-debug-toolbar==3.8.1  # https://github.com/jazzband/django-debug-toolbar
+django-extensions==3.2.1  # https://github.com/django-extensions/django-extensions
+django-coverage-plugin==3.0.0  # https://github.com/nedbat/django_coverage_plugin
+pytest-django==4.5.2  # https://github.com/pytest-dev/pytest-django
```
Comment (CreativePhilip) [inline backend/requirements/requirements.txt]: Lets just set the deps that we use as we go, and not copy paste

**Review by CreativePhilip — CHANGES_REQUESTED** [review]

File: backend/media_server/Dockerfile (hunk 1 above)
Comment (olgaszatkowska) [reply backend/media_server/Dockerfile]: no

File: backend/requirements/requirements.txt (hunk 2 above)
Comment (olgaszatkowska) [reply backend/requirements/requirements.txt]: k

**Comment by CreativePhilip** [comment]
Please delete the maintenance directory and you can merge

File: backend/postgres/maintenance/_sourced/constants.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+
+BACKUP_DIR_PATH='/backups'
+BACKUP_FILE_PREFIX='backup'
```
Comment (CreativePhilip) [inline backend/postgres/maintenance/_sourced/constants.sh:5]: Pls delete this whole maintenance directory and you can merge

**Review by CreativePhilip — APPROVED** [review]

**Pull request merged**